TracksDemo/TracksDemo Mac/TracksDemo Mac.entitlements

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>IDEDidComputeMac32BitWarning</key>
+	<key>com.apple.security.app-sandbox</key>
```

Comment:
Because of the `.plist` structure, Git considers this a rename, but it's actually two actions:

- Add a `.entitlements` for the Mac demo app to enable sandboxing
- Remove the `IDEWorkspaceChecks.plist` file

I wonder why the checks file was removed 🤔 Is this something you did intentionally, or something Xcode automated away?

Reply:
> `Package.swift` now serves as the project structure

I assume to open the project in this configuration you either open `Package.swift` or the entire project folder with Xcode. Right?

In such a case, how do you run the demo app from the Package version of the project in Xcode? 🤔 I tried adding a new scheme for it, but couldn't find a way. 

![image](https://user-images.githubusercontent.com/1218433/133183715-b423cf25-3aac-4bab-9da1-a49a13a8830b.png)

The only way I found was to open the `xcodeproj` in the `TracksDemo` folder. When I do that, Xcode regenerates the IDE checks file.

Reply:
Ah, yep. Opening the .xcodeproj file directly is the expected way to run the demo project; the `Package.swift` file doesn't reference it at all. (Which makes sense, because users of the library don't need to know about it.)

If you want to just open the library code, you can double-click on the `Package.swift` file, and it will open the whole Swift Package structure in Xcode.

Reply:
I don't object to checking in the IDEChecks file for the demo project, though.